packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -5,8 +5,10 @@ import ReactDOM from 'react-dom';
 import ReactDOMServer from 'react-dom/server';
 // eslint-disable-next-line import/no-unresolved
 import ShallowRenderer from 'react-test-renderer/shallow';
+import { version as testRendererVersion } from 'react-test-renderer/package';
```

Comment:
This is no `react-test-renderer/package` in the latest [`react-test-renderer`](https://github.com/facebook/react/tree/master/packages/react-test-renderer)